tests/pyccel/scripts/runtest_degree_in.py

```diff
@@ -3,7 +3,6 @@
 
 @pure
 @types('int')
-@stack_array('tmp')
```

Comment:
Why did you remove this line? The length of a stack array does not need to be known at compile time, neither in Fortran nor in C.

Reply:
Ah, you are right. This doesn't work for 2 reasons in C:
1. The C code uses `Variable.shape` instead of `Variable.alloc_shape`
2. The C code uses compound literals to manage stack arrays. But the size of a compound literal must be known at runtime (it is the error from this which confused me)

Reply:
Actually I think the compound literal is just done wrong. As we know the rank we should know the size. I will open an issue

Reply:
Fixed in ac444f0
The issue is #760

Reply:
Thanks!